agent/README.md

```diff
@@ -358,6 +358,9 @@ You can run the testwise agent in three different modes, configured via the opti
   This is most useful when running tests in a CI/CD pipeline where the build tooling can later batch-convert all `*.exec` files and upload a testwise coverage report to Teamscale or in situations where the agent must consume as little memory and CPU as possible and thus cannot convert the execution data to a report as required by the other options.
   It is, however, less convenient as you have to convert the `*.exec` files yourself.
 
+- `json-file`: The agent stores the coverage in JSON `*.json` files within the `out` directory.
+  This is most useful when running tests in a CI/CD pipeline where the testwise coverage is to be provided in artifact stores, such as S3 or Artifactory, using custom upload scripts.
```

Comment:
Does it make sense to name artifactory here as we have a integrated upload to artifactory within the agent?

The ticket specified the option as `disk`. Did you have a good reason to deviate from that suggestion?

Reply:
Good points. Now the option is called `disk` and I have rephrased the docs.